newsroom/wire/views.py

```diff
@@ -142,6 +142,11 @@ def search():
     return send_response('wire_search', response)
 
 
+@blueprint.route('/copyright')
```

Comment:
I think it would make sense to have another blueprint for this, it's not clear why it's part of wire